pages/install/conda-install.rst

```diff
@@ -45,32 +47,87 @@ obtained are specified by adding the ``--channel`` option in the ``install`` or
 ``cantera`` channel, so we add ``--channel cantera`` and to tell Conda to look at the
 ``cantera`` channel in addition to the default channels.
 
-To use the scripts and modules installed in the ``spam`` environment,
-you must activate it it by running::
+To use the scripts and modules installed in the ``ct-env`` environment, including Jupyter,
+you must activate it it by running:
+
+.. code:: shell
+
+   conda activate ct-env
+
+**Option 2: Create a new environment using an environment file**
+
+This option is similar to **Option 1** but includes a few other packages that
+you may find helpful as you're working with Cantera. Copy and paste the contents
+of the file shown below into a file called ``environment.yaml``. Then, save the
+the file somewhere and remember that location.
+
+.. code:: yaml
+
+   name: ct-env
+   channels:
+   - cantera
+   - defaults
+   dependencies:
+   - python
```

Comment:
Similarly, pointing out that python 3.6 and up (with default option) may be an opportunity for documentation.

Reply:
I'd rather that people install the most recent version of Python that we support, and not specify a particular version here. IMO, the more we can push people to quickly upgrade versions of Python, the more features from recent versions we can use. I can add a comment like:

```yaml
- python  # Cantera supports Python 3.5 and up
```

which is true for 2.5.1.

Reply:
I'd still 'recommend' 3.6 and up, but that's minor.

Reply:
I take that comment back, we don't have conda packages for Python 3.5, see https://anaconda.org/Cantera/cantera/files?version=2.5.1 So I'll change that to say "3.6 and up"